packages/synapse-react-client/src/lib/containers/row_renderers/Dataset.tsx

```diff
@@ -63,6 +73,20 @@ class Dataset extends React.Component<DatasetProps, never> {
       ['SIZE', fileSize],
       ['FILES', fileCount],
     ]
+    if (genericCardSchema && genericCardSchema.secondaryLabels) {
```

Comment:
Originally, FUNDER, SIZE, and FILES were the hard-coded secondary values for Datasets.  Now we add the ability to define your own in the configuration (no special rendering of the value today)